sessionize/app/src/main/res/layout/item_sponsor_pic_small.xml

```diff
@@ -3,4 +3,5 @@
               android:layout_width="80dp"
               android:layout_height="80dp"
            android:layout_margin="10dp"
+           android:background="@color/white"
```

Comment:
Does iOS have the same issue?

Reply:
It's hard to see in iOS but I've updated it anyway to match.